backend/src/services/model.ts

```diff
@@ -319,6 +322,18 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   if (modelDiff.settings?.mirror?.destinationModelId && modelDiff.settings?.mirror?.sourceModelId) {
     throw BadReq('You cannot select both mirror settings simultaneously.')
   }
+  if (modelDiff.collaborators) {
+    const invalidUsers = []
+    modelDiff.collaborators.forEach(async (collaborator) => {
```

Comment:
We can't await the promises from this async callback when using `.forEach()` as it doesn't return anything. Instead we can use `.map()` like so:
```
await Promise.all(modelDiff.collaborators.map(async (collaborator) => { ... }))
```